BLaurenB/techtime
# Style Flash Messages

#### What's this PR do?
Uses bootstrap's built in flash message styling by changing flash keys. Adds alert class in the application html. Removes flash.scss from project.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
app/views/layout/application.html.erb to see the alert css added.

#### How should this be manually tested?
start server, add a freelancer to the cart and the message should be in blue at the top. visit the cart and remove a freelancer and the message should be green. log in and the message should be green in the nav bar.

#### Any background context you want to provide?
#### What are the relevant tickets?
Closes #107 
#### Screenshots (if appropriate)
#### Questions:
